src/main/java/teammates/ui/controller/InstructorCourseRemindAction.java

```diff
@@ -125,6 +139,22 @@ private String extractStudentRegistrationKey(StudentAttributes student) {
         return joinLink.substring(startIndex);
     }
 
+    private String isolatePrevPageUrl() {
```

Comment:
Not too sure if this is the best way. You could just pass in `Const.ActionURIs.INSTRUCTOR_COURSE_DETAILS_PAGE` or `Const.ActionURIs.INSTRUCTOR_STUDENT_LIST_PAGE` and check if the strings occur in the url.

Reply:
You are right. I didn't realize this earlier. I am implementing this like you suggested and will cater to the changes you've pointed out above in the process. Thank you. :)